src/api-service/__app__/onefuzzlib/workers/scalesets.py

```diff
@@ -150,7 +151,7 @@ def set_failed(self, error: Error) -> None:
     def init(self) -> None:
         from .pools import Pool
 
-        logging.info("scaleset init: %s", self.scaleset_id)
+        logging.info(SCALESET_LOG_PREFIX + "init. scaleset_id:%s", self.scaleset_id)
```

Comment:
Is there a reason to concat using `+` vs. going all-in on interpolation? Example of what I mean:
```suggestion
        logging.info("%s init. scaleset_id:%s", SCALESET_LOG_PREFIX, self.scaleset_id)
```
(removing the trailing space from the prefix constant)

Reply:
We use this method in a few places already, as such thought it would be the least impact.

Reply:
My preference would be to use a context scoped logging impl, something on top of opentelemetry.  Once it's available in public preview in Azure, we should move to it.